docs/source/install.rst

```diff
@@ -127,9 +129,9 @@ The installation instructions vary depending on your package manager:
 
       First, set up the `Guix-science channel <https://codeberg.org/guix-science/guix-science>`_. Then, in a terminal, run
 
-      .. code-block:: bash
+        .. code-block:: bash
 
-	  guix install python-cocotb
+	        guix install python-cocotb
```

Comment:
That's unintended?

Reply:
I caught this in the other PR and when I moved that commit here this fix was still in. There was a tab and the indentation was off and IIRC it was causing issue with Sphinx.